src/Workspaces/VisualBasic/Portable/Extensions/SyntaxTreeExtensions.vb

```diff
@@ -139,6 +155,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions
         End Function
 
         <Extension()>
+        Public Function IsBeforeFirstToken(syntaxTree As SyntaxTree, position As Integer, cancellationToken As CancellationToken) As Boolean
```

Comment:
This appears to be language-agnostic, so it might be preferable to put it in a shared helper class.